### PR TITLE
chore: updating packages and pnpm

### PR DESCRIPTION
**Pull request by reynan-dev**
## Ticket

**Title:** ETQ Dev, j’ai des packages updated
**Notion Link:** https://www.notion.so/jkergal/ETQ-Dev-j-ai-des-packages-updated-926d7c9d6fa24bba9bd958948512b5ee

## Description

- Updated pnpm
- Updated packages in the root, server, and client folders
- Created jest.config on the client side
- Created a tests folder on the client side and moved the test to

## Guidelines

🔗 Link your PRs to their related Notion tickets
✍️ Use the commit convention to your pull request title
📃 If your code affects the build, update `README.md`
🚫 Do not merge a PR until all comments are resolved
🗑 Remove your branch after merging

### CODE REVIEW

**Review by jkergal — APPROVED** [review]

File: client/Dockerfile [hunk 1]
```diff
@@ -1,6 +1,6 @@
 FROM node:16-alpine AS base
 
-RUN npm i -g pnpm
+RUN npm i -g pnpm --latest
```
Comment (jkergal) [inline client/Dockerfile:3]: <3

File: client/tsconfig.json [hunk 2]
```diff
@@ -17,5 +17,6 @@
 		"jsx": "react-jsx",
 		"types": ["node", "jest"]
 	},
-	"include": ["src"]
+	"include": ["src"],
+	"exclude": ["node_modules", "dist", "**.test.ts", "**.test.tsx"]
```
Comment (jkergal) [inline client/tsconfig.json:21]: Should we exclude tests for typescript?

**Review by jkergal — APPROVED** [review]
<3

File: client/tsconfig.json (hunk 2 above)
Comment (reynan-dev) [reply client/tsconfig.json:21]: I dont know how to works tests on client side manage by react, i just find it in a web article.

**Pull request merged**